### PR TITLE
waon: init at 0.11

### PR DESCRIPTION
**Pull request by puckipedia**
###### Motivation for this change
I recently had some use for this tool, so I decided to package it :)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by edef1c — APPROVED** [review]

File: pkgs/applications/audio/waon/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ fftw gtk2 libao libsamplerate libsndfile ncurses ];
+
+  installPhase = ''
+    install -Dt $out/bin waon pv gwaon
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A Wave-to-Notes transcriber";
+    homepage = https://kichiki.github.io/WaoN/;
```
Comment (mmahut) [inline pkgs/applications/audio/waon/default.nix]: Please use quotas.

File: pkgs/applications/audio/waon/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    license = licenses.gpl2;
+    maintainers = [ maintainers.puckipedia ];
+    platforms = platforms.unix;
```
Comment (mmahut) [inline pkgs/applications/audio/waon/default.nix]: Does it build only on this platforms?

**Comment by mmahut** [comment]
@GrahamcOfBorg build waon

File: pkgs/applications/audio/waon/default.nix (hunk 2 above)
Comment (puckipedia) [reply pkgs/applications/audio/waon/default.nix]: The author seems to have ran it on MinGW;  I'll drop the platform requirement (it's limited to places that GTK2 can run either way)

File: pkgs/applications/audio/waon/default.nix (hunk 1 above)
Comment (edef1c) [reply pkgs/applications/audio/waon/default.nix]: We don't have a style guide requirement for this, and I'm yet to see anyone calling for deprecation of this syntax. At most, I'd drop the trailing slash for style reasons.

Comment (puckipedia) [reply pkgs/applications/audio/waon/default.nix]: Why? There are many other places that use non-quoted URLs, the style guide does not say it is disallowed, and it seems very weird to not use the feature if it exists.

**Pull request merged**